Lethal-Lemurs/nodejs-bangazon-site
# Db ss payment to order

## Status
**READY**

## Description
The user can now complete an order by closing it and adding a payment type.

## Resolves Issue Number
#4  

## TEST
- drop migrations
- fetch branch to test
- migrate and seed
- login and navigate to to product details and add a product to your order. Make sure that you have already added a payment type or it won't work because you won't have a method to pay. Select your payment method and complete the order. It will close it and add a payment type to your database in psql.


## Impacted Files in Application
    modified:   controllers/ProductDetailsCrtl.js
    modified:   routes/ProductDetailsRoute.js
    modified:   models/order.js
